krun/vm_defs.py

```diff
@@ -318,7 +317,7 @@ def __eq__(self, other):
 
     def parse_instr_stderr_file(self, file_handle):
         """Parse instrumentation data. Override as necessary"""
-        pass
+        return {}
```

Comment:
A quick grep shows that we have used both `{}` and `dict()` to construct empty dictionaries. It isn't important for this PR, and it isn't really worth a PR of its own, but we should probably settle on something to use in future.